src/executables/unload_schema.c

```diff
@@ -1122,7 +1141,14 @@ emit_schema (print_output & output_ctx, DB_OBJLIST * classes, int do_auth, DB_OB
 	  continue;
 	}
 
-      output_ctx ("CREATE %s %s%s%s", is_vclass ? "VCLASS" : "CLASS", PRINT_IDENTIFIER (name));
+      class_name_copy = strdup (name);
+      owner_name_p = strtok_r (class_name_copy, ".", &save_token);
+      class_name_p = strtok_r (NULL, ".", &save_token);
+
+      output_ctx ("CREATE %s %s%s%s.%s%s%s", is_vclass ? "VCLASS" : "CLASS", PRINT_IDENTIFIER (owner_name_p),
+		  PRINT_IDENTIFIER (class_name_p));
+
+      free_and_init (class_name_copy);
```

Comment:
Please refer to the code below
```
      char owner_name_p[512]; 
      strcpy(owner_name_p, name);
      class_name_p = strtchr (owner_name_p, '.');
      *class_name_p++ = '\0';

      output_ctx ("CREATE %s %s%s%s.%s%s%s", is_vclass ? "VCLASS" : "CLASS", PRINT_IDENTIFIER (owner_name_p),
		  PRINT_IDENTIFIER (class_name_p));
```

Reply:
I will use this method while changing to a macro.